cmd/backup-manager/app/backup/manager.go

```diff
@@ -161,7 +153,8 @@ func (bm *BackupManager) performBackup(backup *v1alpha1.Backup, db *sql.DB) erro
 	}
 	glog.Infof("get cluster %s commitTs %s success", bm, commitTs)
 
-	bucketURI := bm.getDestBucketURI()
+	remotePath := strings.TrimPrefix(archiveBackupPath, constants.BackupRootPath+"/")
```

Comment:
s/increase/set is better in line 97

Reply:
RestTikvGCLifeTimeFailed typo in line 115

Reply:
should return err when GetTikvGCLifeTimeFailed? I found bm.setTikvGClifeTime(db, oldTikvGCTime) logic will use oldTikvGCTime.

Reply:
When `getTikvGCLifeTimeFaile` failed,  the status of backup CR will be updated and then returned. refer https://github.com/pingcap/tidb-operator/blob/master/cmd/backup-manager/app/backup/manager.go#L97-L102

Reply:
OK I don't notice return before updater.